store/store.go

```diff
@@ -170,7 +174,7 @@ func assertsURL() string {
 }
 
 func myappsURL() string {
-	if os.Getenv("SNAPPY_USE_STAGING_MYAPPS") != "" {
+	if os.Getenv("SNAPPY_USE_STAGING_CPI") != "" {
```

Comment:
To make it easy to switch to staging server. It doesn't make sense to use staging CPI but prod MyApps (ie. things will not work ok). myappsURL is the base url from which macaroons are got to interact with CPI. I can undo this change, if you think it was better before.